### DISCUSSION TITLE
setting up github invitation bot

### DISCUSSION THREAD

**Discussion by Adarsh-jaiss** [TechStudent10/techstudent10#40] [Q&A] [answered]
hey, i have created workflow and addded security token , but still my github invitation bot isn't working?
can you please help me to fix this ?
i am using this : https://github.com/marketplace/actions/invite-on-label

**Comment by TechStudent10** [comment]
Hello Adarsh.

Can you send me the repo you are trying this on?

**Reply by Adarsh-jaiss** [reply]
yeah
just a second

**Reply by Adarsh-jaiss** [reply]
https://github.com/collab-community/become-a-member

**Comment by TechStudent10** [comment]
@Adarsh-jaiss did you add the `INVITE_TOKEN` personal access token with `admin:org` permissions with `No expiration`, then add it in the `Organization Settings > Secrets`?

**Reply by Adarsh-jaiss** [reply]
yes

**Reply by Adarsh-jaiss** [reply]
but still it isnt working

**Comment by TechStudent10** [comment, marked as answer]
@Adarsh-jaiss You misspelled "organization", kinda. In the workflow, you use US spelling (organization) while in the issue itself, you are using UK spelling (organisation) the difference is the "z" in the US spelling is switched with an "s" in the UK spelling.

**Reply by Adarsh-jaiss** [reply]
oh okay
so, i need to use organization

**Reply by TechStudent10** [reply]
I have created a PR

**Reply by Adarsh-jaiss** [reply]
yeah
merged it

**Reply by Adarsh-jaiss** [reply]
kindly check the bot , is it working or not

**Reply by TechStudent10** [reply]
Okay I'll run another test (the one I just did doesn't count)

**Reply by Adarsh-jaiss** [reply]
yeah, its working

**Reply by TechStudent10** [reply]
Okay! It works!

![image](https://user-images.githubusercontent.com/76978184/181812426-59c911e8-5309-4cff-90fa-bf44097998a8.png)

**Reply by Adarsh-jaiss** [reply]
Thank you so much 
btw, what's your name if i may ask

**Reply by TechStudent10** [reply]
My real name? Not saying.
